talk/morelanguage/morestl.tex

```diff
@@ -22,6 +22,39 @@ \subsection{More STL}
   \end{exampleblock}
 \end{frame}
 
+\begin{frame}[fragile]
+  \frametitlecpp[20]{\texttt{std::span<T>}}
+  \begin{block}{Non owning view of a continuous sequence of items}
+    \begin{itemize}
+    \item Doesn't allocate memory
+    \item Allows to modify items, but not the storage itself
+      \begin{itemize}
+      \item no reallocation, push or pop
+      \end{itemize}
+    \item Provides iterators, \mintinline{cpp}{size}, \mintinline{cpp}{front}/\mintinline{cpp}{back}, \mintinline{cpp}{subspan}, ...
+    \item Standard contiguous containers convert to it automatically
+      \begin{itemize}
+      \item \mintinline{cpp}{std::vector}, \mintinline{cpp}{std::array}, C arrays, ...
```

Comment:
Should we mention that non-contiguous containers like `std::list`, `std::deque` etc. don't convert?